.github/workflows/reusable_ragger_tests.yml

```diff
@@ -103,11 +103,20 @@ jobs:
           ref: ${{ inputs.app_branch_name }}
           submodules: recursive
 
+      - name: Set binary path
+        run: |
+          if [[ "${{ needs.call_get_app_metadata.outputs.is_rust }}" == "true" ]];
+          then
+              echo "BINARY_PATH=${{ needs.call_get_app_metadata.outputs.build_directory }}/target/" >> $GITHUB_ENV
+          else
+              echo "BINARY_PATH=${{ needs.call_get_app_metadata.outputs.build_directory }}/build/" >> $GITHUB_ENV
+          fi
+
       - name: Download app binaries
         uses: actions/download-artifact@v3
         with:
           name: ${{ inputs.download_app_binaries_artifact }}
-          path: ${{ needs.call_get_app_metadata.outputs.build_directory }}/build/
+          path: ${{ env.BINARY_PATH }}
```

Comment:
I prefer the `step.outputs.var` syntax over `env.var` because it's more explicit but that's my preference